src/Actions/Ticket/UpdateTicket.php

```diff
@@ -35,7 +38,22 @@ public function performAction(): Model
         $ticket->save();
 
         if (is_array($users)) {
-            $ticket->users()->sync($users);
+            $sync = $ticket->users()->sync($users);
```

Comment:
no need to create extra variable, can just put this directly into the data_get() as in CreateTicket action.